Services/CO.CDP.Localization/FormsEngineResource.resx

```diff
@@ -240,6 +240,127 @@
   <data name="Qualifications_Configuration_FurtherQuestionsExemptedHeading" xml:space="preserve">
     <value>Do you want to add any relevant qualifications?</value>
   </data>
+
+  <!-- Steel -->
+
+  <!-- Section Info -->
+  <data name="Steel_SectionTitle" xml:space="preserve">
+    <value>Steel module</value>
+  </data>
+  <data name="Steel_Configuration_FurtherQuestionsExemptedHint" xml:space="preserve">
+    <value>You'll need to provide information on Steel Specific supply chain projects or programmes.This may include relevant examples of: </value>
+  </data>
+  <data name="Steel_Configuration_FurtherQuestionsExemptedHeading" xml:space="preserve">
+    <value>The steel information you'll need</value>
+  </data>
+  <data name="Steel_Configuration_AddAnotherAnswerLabel" xml:space="preserve">
+    <value>Add another Steel Information?</value>
+  </data>
+  <data name="Steel_Configuration_SingularSummaryHeading" xml:space="preserve">
+    <value>You have added Steel Information</value>
+  </data>
+  <data name="Steel_Configuration_RemoveConfirmationHeading" xml:space="preserve">
+    <value>Are you sure you want to remove this Steel Information?</value>
+  </data>
+  <data name="Steel_Configuration_PluralSummaryHeadingFormat" xml:space="preserve">
+    <value>You have added {0} Steel Information</value>
+  </data>
+  <data name="Steel_Configuration_SingularSummaryHeadingHint" xml:space="preserve">
+    <value>Steel module</value>
+  </data>
+  <data name="Steel_Configuration_PluralSummaryHeadingHintFormat" xml:space="preserve">
+    <value>Steel module</value>
```

Comment:
I'm not a content designer, but maybe this should be called "Steel Information" since that's how it's referred to elsewhere? Doesn't feel like the word "module" means much.

```suggestion
    <value>Steel Information</value>
```